ICAFIX/hcp_fix_multi_run

```diff
@@ -103,10 +108,12 @@ opts_AddMandatory '--concat-fmri-name' 'ConcatName' 'run1_to_3' "Name (including
 
 opts_AddOptional '--motion-regression' 'doMotionRegression' 'TRUE or FALSE' "Controls whether to regress motion parameters as part of the cleaning, default FALSE" 'FALSE'
 
-opts_AddOptional '--training-file' 'TrainingData' 'somefile.RData' 'Specify a custom training file, defaults to "HCP_Style_Single_Multirun_Dedrift.RData"' ''
+opts_AddOptional '--training-file' 'TrainingData' 'somefile.RData, somefile.pyfix_model, or a pyfix built-in model without extension' 'Specify a custom training file, defaults to "HCP_Style_Single_Multirun_Dedrift.RData"' ''
 
 opts_AddOptional '--fix-threshold' 'FixThresh' 'number' 'Threshold setting in FIX that controls the sensitivity/specificity tradeoff, default = 10' '10'
 
+opts_AddOptional '--T1wTemplateBrain' 'T1wTemplateBrain' 'somefile.nii.gz' "path to T1w template brain, used by pyfix" "${HCPPIPEDIR}/global/templates/MNI152_T1_1mm_brain.nii.gz"
+
```

Comment:
Typically we use template brains that are matched to the structural resolution.  Should that be done in this case? What is this currently being used for?

Reply:
It is used for the creation of vein masks by pyfix. The template is copied to ${concatfmrihp}.ica/reg/standard.nii.gz.  If there is no {concatfmrihp}.ica/reg/standard.nii.gz pyfix will create one with 1 mm resolution. While I added the parameter to accommodate NHP. I believe you are correct that that the default should be the 0.7mm for HCP data. I have change this added a not about matching resolutions in the parameter description.

Reply:
Is this required for human processing or just macaque processing?